src/Extension/Task/DeprecationsCheckTask.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 id: "validation:php:deprecations-check"
-short_description: "Check your code by running a static code analysis and generate report with deprecations"
-help: ~
+short_description: "Check your code by running a static code analysis and generate report with deprecations. By default uses 'psalm.xml' config file."
+help: "In case of critical errors task exits with code '1'. You can change it by specifying 'errorLevel=1' in the 'psalm.xml'."
```

Comment:
This is misleading and could be understood as the exit code can be adjusted here.
Instead, it should be made clear that the error level is about the introspection, e.g.

> You can adjust the depth of analysis by specifying `errorLevel=[1-8]` in `psalm.xml` file.